src/bindings/python/src/openvino/frontend/pytorch/ts_decoder.py

```diff
@@ -167,8 +169,12 @@ def _get_scripted_model(self, pt_module, example_inputs=None, skip_freeze=False,
                 if trace_kwargs is None:
                     trace_kwargs = {}
                 try:
-                    scripted = torch.jit.trace(
-                        pt_module, **input_parameters, strict=False, **trace_kwargs)
+                    with BuiltinPatcher():
+                        # patch Python environment for the short time
+                        # it patches built-in functions such as divmod()
+                        # which TorchScript is uncapable to trace for torch.Tensor input type
```

Comment:
Lets move this comment to the inside of patcher code. Decoder is already complicated, I would prefer to reduce the unnecessary information there

Reply:
Moved